extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/FastBootEntityManagerFactoryBuilder.java

```diff
@@ -76,7 +80,8 @@ public EntityManagerFactory build() {
         try {
             final SessionFactoryOptionsBuilder optionsBuilder = metadata.buildSessionFactoryOptionsBuilder();
             populate(persistenceUnitName, optionsBuilder, standardServiceRegistry);
-            return new SessionFactoryImpl(metadata, optionsBuilder.buildOptions());
+            return new SessionFactoryImpl(metadata, optionsBuilder.buildOptions(),
+                    metadata.getTypeConfiguration().getMetadataBuildingContext().getBootstrapContext());
```

Comment:
Is this really necessary? I vaguely remember removing this because it was deprecated.

Reply:
The  SessionFactoryImpl constructor with 2 parameters is deprecated, I used the recommended one.

Reply:
Well `getMetadataBuildingContext` is deprecated too, so this doesn't really solve the problem.

Anyway, even though it's a bit off-topic for this PR, I guess we can keep this since it's not worse than it was before.